samples/crypto/ecdh/README.rst

```diff
@@ -7,31 +7,28 @@ Crypto: ECDH
    :local:
    :depth: 2
 
+The ECDH sample shows how to perform an Elliptic-curve Diffie–Hellman key exchange to allow two parties to obtain a common secret.
```

Comment:
Please change "common secret" to "shared secret" as this is  statement that is standardized phrase in Diffie-Hellman key exchange .